packages/react-liveness/src/components/FaceLivenessDetector/service/machine/machine.ts

```diff
@@ -60,6 +61,7 @@ import { WS_CLOSURE_CODE } from '../utils/constants';
 
 const CAMERA_ID_KEY = 'AmplifyLivenessCameraId';
 const DEFAULT_FACE_FIT_TIMEOUT = 7000;
+const ALTERNATE_MIME_TYPE = 'video/x-matroska;codecs=vp8';
```

Comment:
Is this supported in safari?

Reply:
Seems like certain versions support it https://caniuse.com/?search=vp8

but we might not be going with this solution anyways